pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/Patch.java

```diff
@@ -161,30 +185,67 @@ public static boolean patch(@NonNull String id, @NonNull String user, @NonNull N
         return true;
     }
 
-    private static String toErrorMessage(String table, String column, Object value, String primaryKey, String id) {
-        return String.format("Error updating %s.%s with value %s for %s=%s", table, column, value, primaryKey, id);
-    }
+    /****/
 
-    public static Boolean isValueUnique(@NonNull DBElement dbElement) {
```

Comment:
This was not used

Reply:
Great to see pruning of irrelevant code :)